packages/conveyer/src/Conveyer.ts

```diff
@@ -480,6 +482,11 @@ class Conveyer extends Component<ConveyerEvents> {
     }
     return scrollPos;
   }
+  private _checkNestedMove(e: any, scrollPos: number) {
+    if (scrollPos === 0 || scrollPos === this._scrollSize - this._size) {
```

Comment:
How about using state instead of scrollAreaElement.scrollLeft?
```js
if (this.isReachStart || this.isReachEnd) {
}
```

Reply:
It seems more appropriate to use state for this feature. 🙂